goaviatrix/distributed_firewalling_zero_trust_rule.go

```diff
@@ -0,0 +1,50 @@
+package goaviatrix
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+)
+
+type DistributedFirewallingZeroTrustRule struct {
+	Action  string `json:"action"`
+	Logging bool   `json:"logging"`
+	UUID    string `json:"uuid,omitempty"`
+}
+
+func (c *Client) UpdateDistributedFirewallingZeroTrust(ctx context.Context, request *DistributedFirewallingZeroTrustRule) error {
+	endpoint := "microseg/edit-zero-trust"
```

Comment:
If you can PUT on this endpoint, why can't you GET instead of the searching business below?